stable/mongodb/templates/statefulset-primary-rs.yaml

```diff
@@ -68,7 +68,8 @@ spec:
       {{- if .Values.extraInitContainers }}
 {{ tpl .Values.extraInitContainers . | indent 6}}
       {{- end }}
-      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
+      {{- $needsVolumePermissions := and .Values.volumePermissions.enabled (and ( and .Values.peristence.enabled (not .Values.persistence.existingClaim) ) .Values.securityContext.enabled) }}
```

Comment:
There is a typo here, `peristence` should be `persistence`.

Reply:
Thanks, fixed.